include/AT_DataParticle.h

```diff
@@ -317,7 +317,7 @@ int AT_particle_name_from_particle_no( const long  n,
  * Returns particle names for given particle numbers
  * @param[in]  n              TODO
  * @param[in]  particle_name  particle names
- * @param[out] corresponding  particle numbers
+ * @param[out] particle_no    TODO
```

Comment:
Change to:
```
 * @param[in]  particle_no  type of the particles
 * @see          AT_DataParticle.h for definition
```

Reply:
Should this @see annotation be inserted into AT_DataParticle.h itself?

Reply:
good point, not necessary here